cmd/crank/version/fetch.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2023 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package version contains common functions to get versions
+package version
+
+import (
+	"context"
+	"strings"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Could you import this as `metav1` please. Just to be consistent with what we do elsewhere to avoid collisions with lots of API packages being named `v1`.